### PR TITLE
Fix #7415. Printing Width was not applied to error messages.

### LINKED ISSUE #7415: "Set Printing Width" does not change the width of error messages
#### Version
8.8
#### Operating system
any
#### Description of the problem
Set Printing Width does not change the width of error messages. To reproduce:
```
Coq < Lemma foo: (forall x:bool, x = 1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9) -> False.
Toplevel input, characters 32-139:
> Lemma foo: (forall x:bool, x = 1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9) -> False.
>                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error:
In environment
x : bool
The term
 "1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 +
  2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 +
  3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9" has type
 "nat" while it is expected to have type "bool".

Coq < Set Printing Width 500.
Coq < 
Lemma Lemma foo: (forall x:bool, x = 1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9) -> False.
Proof.
Toplevel input, characters 32-139:
> Lemma foo: (forall x:bool, x = 1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9+1+2+3+4+5+6+7+8+9) -> False.
>                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error:
In environment
x : bool
The term
 "1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 +
  2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 +
  3 + 4 + 5 + 6 + 7 + 8 + 9 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9" has type
 "nat" while it is expected to have type "bool".
```
Making the width smaller also does not change the printing.

### PR DESCRIPTION
**Pull request by Matafou**
**Kind:** bug fix.

Fixes / closes #7415

### CODE REVIEW

**Comment by SkySkimmer** [comment]
Could you summarize the issue in the commit message so that it's not just a reference to github? Thanks.

**Comment by Matafou** [comment]
Sure sorry.

**Review by ejgallego — APPROVED** [review]
Obviously correct, taking this myself as this file does morally belong to `toplevel`
